tests/test_schema.yaml

```diff
@@ -585,6 +585,27 @@ policies:
     - is_owner
   resource:
     path: /v2.0/owned_resource.*
+- action: '__attach__'
+  effect: allow
+  id: tenant_isolation_domain_admin_server_to_network
+  principal: admin
+  relation_property: network_id
+  target_condition:
+  - is_owner
+  - skip_tenant_domain_check
```

Comment:
Why both `is_owner` and `skip_tenant_domain_check`?
It looks like `skip_tenant_domain_check` overrides `is_owner`.

Reply:
no i does not - it does not remove tenant whole tenant isolation it just allows to create resource in different tenant/domain than related resource providing that user has access to it. This is useful for some "public" resources